node/config/types.go

```diff
@@ -50,6 +50,45 @@ type StorageMiner struct {
 	Storage    sectorstorage.SealerConfig
 	Fees       MinerFeeConfig
 	Addresses  MinerAddressConfig
+	DAGStore   DAGStoreConfig
+}
+
+type DAGStoreConfig struct {
```

Comment:
I thought of that, but miners want flexibility here because transients, index and datastore have different sizing expectations, and also require different durability and access guarantees (e.g. transients can be wiped out, but index and datastore must remain consistent and durable). In practice, miners will place transients in volatile/cheap storage, and index and datastore in durable, replicated, fast-access storage.

Granted that miners can use symlinks, but I consider that a hack to overcome a limitation of the system.

> "if anything goes wrong just delete the dagStore directory"

I think this is a valuable property to have for testing, but as we move to production, destroying your dagStore directory won't be something we ask for very frequently (or at all).

Reply:
Ok fair points 👍